File: Source/Zybach.MeanAPI/src/dtos/robust-review-dto.ts
export interface RobustReviewDto {
    wellRegistrationID: string;
    wellTPID?: string;
    lat: number;
    long: number;
    dataSource: string;
    monthlyPumpedVolumeGallons: MonthlyPumpedVolumeGallonsDto[];
}

export interface MonthlyPumpedVolumeGallonsDto {
    month: number;
    year: number;
    volumePumpedGallons: number;
}

File: Source/Zybach.API/Models/RobustReviewDto.cs
using System.Collections.Generic;
using Zybach.Models.DataTransferObjects;

namespace Zybach.API.Models
{
    public class RobustReviewDto
    {
        public string WellRegistrationID { get; set; }
        public string WellTPID { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public string DataSource { get; set; }
        public List<MonthlyPumpedVolume> MonthlyPumpedVolumeGallons { get; set; }
    }
}